dc-cudami-server/dc-cudami-server-backend-jdbi/Scripts/Script.sql

```diff
@@ -0,0 +1,61 @@
+select
```

Comment:
Soll das hier liegen bleiben? Wenn ja, wäre es in `src/sql` mit passender Beschreibung besser aufgehoben.